SkeletonViewCore/Sources/Internal/UIKitExtensions/UITableView+CollectionSkeleton.swift

```diff
@@ -61,6 +61,11 @@ extension UITableView: CollectionSkeleton {
     func removeDummyDataSource(reloadAfter: Bool) {
         guard let dataSource = self.dataSource as? SkeletonCollectionDataSource else { return }
         restoreRowHeight()
+        
+        for cell in self.visibleCells {
```

Comment:
Please, could you provide more details about why this fixes the issue?

Reply:
<img width="1564" alt="screenshot" src="https://user-images.githubusercontent.com/9416042/184343016-037f12e6-7c48-48be-852d-9202d7077b5d.png">

Because here have extra `UITableViewWrapperView` if the style if `groupInseted` and it will block `subview.recursiveHideSkeleton` propagate the call into its tableview cell, so in this fix need call `cell.hideSkeleton()` for each call explicitly

Reply:
Ok, I see your point. In this case, I think we could modify [the method that returns the subviews of a `UITableView`](https://github.com/Juanpe/SkeletonView/blob/fbc2e6f8d22be0dbd4c9e502cd32c9346181dfa1/SkeletonViewCore/Sources/Internal/SkeletonExtensions/SubviewsSkeletonables.swift#L24), something like that:

```swift
if #available(iOS 13.0, *),
   style == .insetGrouped {
         return subviews.first?.subviews ?? []
} else {
        return subviews
}
```

WDYT?

Reply:
It is a good idea, looks like it is more appropriate place compare with `UITableView+CollectionSkeleton.swift`, one suggestion is can we using `visibleCells` instead of the first view? it looks more future-proof if apple change the implementation later, just like below

```swift
return visibleCells.filter { $0.isSkeletonable } + subviews // we need subviews for section header / footer
```

Reply:
Yep, first, the code checked `visibleCells `, but we had problems. Here is the PR that contains the fix (#402 )

Reply:
However, I agree with you that this code couldn't be enough 🤔 because the headers and footers don't belong to the WrapperView.

So, I think the best solution could be to return a transformed array. I mean, check if a subview is a `UITableViewWrapperView` and if so, append its subviews to the subviews of the table. Do you know what I mean?

```swift
var result = [UIView]()
for subview in subviews {
    if String(describing: type(of: subview)) == "UITableViewWrapperView" {
        result.append(contentsOf: subview.subviews)
    } else {
        result.append(subview)
    }
}
return result
```

Reply:
Hi @Juanpe, I have update the PR base on you suggestion, can you take a look to see if is good to you